c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
```

Comment:
I could create a variable BaseURL and have something like

```
    $('a').click(function() {
        $(this).attr('route') ? window.location.href = BaseURL/route :  window.location.href = BaseURL;
    });
```
Angular ng-href would be nice... ;p